healthcare/datasets/datasets.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {google} = require('googleapis');
+
+// [START healthcare_create_dataset]
+function createDataset (client, projectId, cloudRegion, datasetId) {
+  // Client retrieved in callback
+  // getClient(serviceAccountJson, function(client) {...});
+  // const cloudRegion = 'us-central1';
+  // const projectId = 'adjective-noun-123';
+  // const datasetId = 'my-dataset';
+  const parentName = `projects/${projectId}/locations/${cloudRegion}`;
+
+  const request = {parent: parentName, datasetId: datasetId};
+
+  client.projects.locations.datasets.create(request)
+    .then(() => {
+      console.log(`Created dataset: ${datasetId}`);
+    })
+    .catch(err => {
+      console.log('ERROR:', err);
+    });
+}
+// [END healthcare_create_dataset]
+
+// [START healthcare_delete_dataset]
+function deleteDataset (client, projectId, cloudRegion, datasetId, cb) {
+  // Client retrieved in callback
+  // getClient(serviceAccountJson, function(client) {...});
+  // const cloudRegion = 'us-central1';
```

Comment:
Can you use default parameters instead of these comments? We usually don't use `adjective-noun-123` but `PROJECT_ID` with a TODO for the user to replace it..